libminifi/include/io/tls/TLSServerSocket.h

```diff
@@ -53,7 +53,7 @@ class TLSServerSocket : public BaseServerSocket, public TLSSocket {
   /**
    * Registers a call back and starts the read for the server socket.
    */
-  void registerCallback(std::function<bool()> accept_function, std::function<int(std::vector<uint8_t>*, int *)> handler);
+  void registerCallback(std::function<bool()> accept_function, std::function<int(std::vector<uint8_t>*, int *)> handler, uint16_t timeout = 3000);
```

Comment:
using `std::chrono::milliseconds` would be clearer than `uint16_t`

Reply:
Fixed in [e35f799](https://github.com/apache/nifi-minifi-cpp/pull/959/commits/e35f799294095be494056f415a24b7350192e61d)